weblate/formats/models.py

```diff
@@ -24,6 +24,28 @@
 from weblate.utils.classloader import ClassLoader
 
 
+class ExporterLoader(ClassLoader):
+    def __init__(self):
+        super().__init__("WEBLATE_EXPORTERS", False)
+        self.errors = {}
+
+
+EXPORTERS = ExporterLoader()
+
+
+def get_exporter(name):
+    """Return registered exporter."""
+    return EXPORTERS[name]
+
+
+def list_exporters(translation):
```

Comment:
Can you please make this a method of ExporterLoader?

Reply:
Ok.